src/app/page.js

```diff
@@ -1,14 +1,27 @@
+"use client";
 import Image from "next/image";
-import styles from "./page.module.css";
 import LoginForm from "../components/login-form";
+import Navbar from "@/components/Navbar";
 
 export default function Home() {
   return (
-    <main className={styles.main}>
-      <h1>Welcome to Koalibook</h1>
-      <div className={styles.loginContainer}>
-        <h3>Login</h3>
-        <LoginForm />
+    <main>
+      <Navbar />
+      <div className="container">
+        <h1>Welcome to Koalibook</h1>
+        <h3>Your Cozy Space to Manage Your Reading Journey!</h3>
```

Comment:
If I am being picky, you should not skip a number in the headings for accessibility reasons. Your h3 should be a h2 actually. You can style your h2 exactly as a h3 would look like :) 
I found this piece of information about it: "People who use screen readers (assistive software that converts text to audio or braille) often scan through subheadings to find the information they need. When subheadings appear in the wrong order, these users may believe that they’re missing out on important content."